Gordon360/Models/CCT/Context/efpt.CCT.config.json

```diff
@@ -8,6 +8,7 @@
    "ModelNamespace": null,
    "OutputContextPath": "Models\/CCT\/Context",
    "OutputPath": "Models\/CCT",
+   "PreserveCasingWithRegex": true,
```

Comment:
Why did this setting change? Can you explain what it does? I don't think I'm familiar with it.

Reply:
I'm not actually sure. I did not actively make this change.

Reply:
Reverted changes